sdk/storage/Azure.Storage.Queues/src/Models/QueueItem.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.Storage.Queues.Models
+{
+    /// <summary>
+    /// QueueItem.
+    /// </summary>
+    public partial class QueueItem
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        internal QueueItem() { }
+
+        /// <summary>
+        /// The name of the Queue.
+        /// </summary>
+        [CodeGenMember("Property")]
```

Comment:
Do these attributes spare us writing custom mapping logic?

Reply:
This makes it so we don't need to do this in a readme transform.